scripts/comfyui.py

```diff
@@ -1,52 +1,124 @@
+import functools
+import json
+from pathlib import Path
 import gradio as gr
+import torch
 
 import modules.scripts as scripts
-from lib_comfyui import webui_callbacks, webui_settings, global_state, platform_utils
-from comfyui_custom_nodes import webui_postprocess_input, webui_postprocess_output
+from lib_comfyui import webui_callbacks, webui_settings, global_state, platform_utils, external_code
 from lib_comfyui.polling_client import ComfyuiNodeWidgetRequests
-from modules import shared, ui
+from modules import shared, ui, sd_samplers
+
+
+def add_default_workflows():

```

Comment:
move this to a different file. in fact, split this entire file into smaller files inside lib_comfyui